GliaWidgetsObjC/README.md

```diff
@@ -0,0 +1,23 @@
+## Objective-C wrapper for [GliaWidgets](https://github.com/salemove/ios-sdk-widgets)
+
+GliaWidgetsSDK developed used pure Swift and doesn't provide Objective-C compatible interfaces. This sample Xcode project shows possibility to get full compatibility with Objective-C.
+
+##### NB! This project is not updating with latest released GliaWidgetsSDK versions and shows how to get Objective-C interfaces only.
```

Comment:
```suggestion
**NB! This project is not kept up to date with latest GliaWidgetsSDK releases, is only shows how to get Objective-C interfaces.**
```

Reply:
**NB! This project is not kept up to date with latest GliaWidgetsSDK releases. It only shows how to get Objective-C interfaces.**